community/modules/scheduler/schedmd-slurm-gcp-v5-controller/main.tf

```diff
@@ -27,12 +27,16 @@ locals {
   # Also, slurm imposed a lot of restrictions to this name, so we format it to an acceptable string
   tmp_cluster_name   = substr(replace(lower(var.deployment_name), "/^[^a-z]*|[^a-z0-9]/", ""), 0, 8)
   slurm_cluster_name = var.slurm_cluster_name != null ? var.slurm_cluster_name : local.tmp_cluster_name
+
+  enable_public_ip_access_config = var.disable_controller_public_ips == false ? [{ nat_ip = null, network_tier = null }] : []
```

Comment:
nit: You could simplify this a littler with:
`= var.disable_controller_public_ips ? [] : [{ nat_ip = null, network_tier = null }]`

Reply:
I thought about this one, in terraform, I have been inclined towards explicit (long) tests, like == false.  But it maybe silly, do you feel strongly about this?

Reply:
I re-read it, and I think you are right, it's more direct and more clear.